glean-core/rlb/src/private/ping.rs

```diff
@@ -15,7 +12,7 @@ pub struct PingType {
 impl PingType {
     /// Creates a new ping type.
     ///
-    /// # Arguments
+    /// ## Arguments
```

Comment:
As per https://mozilla.github.io/glean/book/dev/core/documentation-guidelines.html:

```suggestion
    /// # Arguments
```